program/src/state.rs

```diff
@@ -36,6 +37,8 @@ pub(crate) fn collect_treasury_seeds<'a>() -> [&'a [u8]; 1] {
 }
 
 /// Derive the address of the governance config account.
+// Jon: I think this needs to be a PDA based on the stake config account, since the stake
+// program can have multiple configs
```

Comment:
What does the presence of multiple stake configs mean exactly? Does that mean there might be multiple parallel staking ecosystems? If so, then yeah I guess it makes sense to have multiple parallel governance protocols, one for each staking ecosystem. Otherwise, it felt more secure to me to have one single governing config over this protocol entirely.

Reply:
Yeah exactly. For example, if we reuse the staking program for lockups, there will be two staking configs

Reply:
Done in #10.